tests/_support/Page/Framework.php

```diff
@@ -1128,5 +1128,34 @@ public function iUploadTheItemTypeCASEFile()
 
         return $this;
     }
+    
+    /**
+     * @When /^I download the framework PDF file$/
+     */
+    public function iDownloadTheFrameworkPdfFile(): Framework
+    {
+        $I = $this->I;
+
+        $I->click('Export', '#itemInfo');
+        $I->waitForElementVisible('#exportModal a.btn-export-pdf');
+        $I->click('#exportModal a.btn-export-pdf');
+        $url = $I->grabAttributeFrom('#exportModal a.btn-export-pdf', 'href');
+        $this->filename = $I->download($url);
+
+        return $this;
+    }
+
+    /**
+     * @Then /^I should see content in the PDF file$/
+     */
+    public function iShouldSeeContentInThePDFFile(): Framework
+    {
+        $I = $this->I;
+
+        $PdfContent = $I->pdf2text($this->filename);
+        $I->assertNotEmpty($PdfContent, 'PDF file is empty');
+        $I->assertContains($I->getLastFrameworkTitle(), $PdfContent, 'Exported PDF');
```

Comment:
The displayed error here `Exported PDF` might want to be more explict as to the problem (to make it easier to see when looking at the test output).

Reply:
@anuradhas248 I think you worked on this, can you please modify the error text?